Makefile

```diff
@@ -3,7 +3,7 @@ HOSTNAME=registry.terraform.io
 NAMESPACE=cockroachdb
 NAME=cockroach
 BINARY=terraform-provider-${NAME}
-VERSION=1.6.0
+VERSION=1.9.0
```

Comment:
Oops, Looks like I missed this last time.  Do you know what this controls?

Reply:
Just the other rules in the Makefile here; it was helpful to have it be correct for `make install` when I was testing locally.